src/Evolution/Systems/GrMhd/GhValenciaDivClean/Subcell/NeighborPackagedData.hpp

```diff
@@ -88,12 +88,6 @@ struct NeighborPackagedData {
         typename grmhd::ValenciaDivClean::System::variables_tag;
     using grmhd_evolved_vars_tags = typename grmhd_evolved_vars_tag::tags_list;
 
-    ASSERT(not db::get<domain::Tags::MeshVelocity<3>>(box).has_value(),
-           "Haven't yet added support for moving mesh to DG-subcell. This "
-           "should be easy to generalize, but we will want to consider "
-           "storing the mesh velocity on the faces instead of "
-           "re-slicing/projecting.");
```

Comment:
Add code comment with this possible optimization?

Reply:
I'll add it to the dox since I worry that the code comment will get lost :)